dev_guide/builds.adoc

```diff
@@ -1064,6 +1063,9 @@ $ oc secrets add serviceaccount/builder secrets/scmsecret
 ----
 ====
 
+NOTE: Depending on the `*limitSecretReferences*`, `*serviceAccountConfig*` section
```

Comment:
There's no way to know whether or not this is enabled client side.  Is it really worth calling it out as optional?